### PR TITLE
PLANET-5225: Carousel Header: remove Zoom and slide to gray style

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-5225
Needs: https://github.com/greenpeace/planet4-base-fork/pull/83 for tests to pass

Screenshot 2020-07-17 at 10 08 12

_Choice of Carousel style is no more possible, Full width classic is the only one_

## Modifications

Mostly deletions:
- Specific files to zoom and slide style 
  - `assets/src/blocks/Carouselheader/ZoomAndSlideCarouselHeader.js`
  - `templates/blocks/carousel_header_zoom-and-slide-to-gray.twig`
- Modification of default style in code to full with classic
- Modification of usage report https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/324/files#diff-0c6007e1c5c6397bba7b97cd6808c413R199
  Although I'm not sure how this is used, if somebody has any tips on this ?

## Tests

- Add a new page, add a Carousel to it
- There shouldn't be any choice in the style used.

### CODE REVIEW

File: assets/src/blocks/Carouselheader/CarouselHeader.js [hunk 1]
```diff
@@ -80,26 +80,16 @@ export class CarouselHeader extends Component {
...
-                image: window.p4ge_vars.home + 'images/carousel-classic.png',
-                value: 'full-width-classic',
-                help: __('This is a full width slider with a classic look: big slides, fade transition, and no subheaders.'),
-              },
-            ]}
-          />
+          <div className='LayoutSelector'>
+          <label>
+            <div>{__('Full width classic', 'p4ge')}</div>
+            <img src={window.p4ge_vars.home + 'images/carousel-classic.png'}/>
```
Comment (mleray) [inline assets/src/blocks/Carouselheader/CarouselHeader.js:89]: Maybe you could avoid using concatenation here, sthg like:
```{`${window.p4ge_vars.home}images/carousel-classic.png`}

File: assets/src/blocks/Carouselheader/CarouselHeader.js [hunk 2]
```diff
@@ -80,26 +80,16 @@ export class CarouselHeader extends Component {
...
-                label: __('Full width classic', 'p4ge'),
-                image: window.p4ge_vars.home + 'images/carousel-classic.png',
-                value: 'full-width-classic',
-                help: __('This is a full width slider with a classic look: big slides, fade transition, and no subheaders.'),
-              },
-            ]}
-          />
+          <div className='LayoutSelector'>
+          <label>
+            <div>{__('Full width classic', 'p4ge')}</div>
```
Comment (mleray) [inline assets/src/blocks/Carouselheader/CarouselHeader.js:88]: Not sure we still need to put a name (and a [help text](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/324/files#diff-b1e24ef490bf7dce9f07bdeb1b988867R90)) if that's the only style available? 🤔

File: classes/blocks/class-carouselheader.php [hunk 3]
```diff
@@ -37,7 +37,7 @@ public function __construct() {
 				'attributes'      => [
 					'block_style'       => [
 						'type'    => 'string',
-						'default' => 'zoom-and-slide-to-gray',
+						'default' => 'full-width-classic',
```
Comment (mleray) [inline classes/blocks/class-carouselheader.php:40]: Wouldn't it make sense to just completely remove this attribute if we cannot change it anyway?

**Comment by lithrel** [comment]
@mleray I went to remove block_style attribute, and took a few other things with it 😬  (subheader and onchange attributes)
Also fixed a warning from FocalPointPicker. I kept the help message, it felt more welcoming to have a little description, but I can take it away if you prefer, our editors probably know what they're using. Ready for re-review 👀

**Comment by lithrel** [comment]
> However I see that in [this file](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/f975872a8d793c0bb297cc132bf82a70cd6aa4a4/assets/src/blocks/Carouselheader/CarouselHeaderBlock.js) there are still several `subheader` mentions

My radar is broken, didn't see those 😨 Fixed 🔧 📡

**Pull request merged**